Manual.md

```diff
@@ -781,6 +781,17 @@ with the character `r` in the `version` variable. The `distfiles`
 location will automatically be set as well as the package made to depend
 on `R`.
 
+- `gemspec` For packages that use
+[gemspec](https://guides.rubygems.org/specification-reference/) files, this is necessary
+for ruby projects that provide extensions with compiled code (like C), if your ruby project
+does not provide any of those then consider using the `gem` build style instead.
+
+- `gem` For packages that are installed using gems from [RubyGems](https://rubygems.org/)
+or other providers. Packages that use this build style must follow the ruby-`pkgname`
```

Comment:
So *ruby-* should not be part of pkgname? If it should, `ruby-<name>` is clearer and consistent with _naming conventions_ section.